tests/unit/resizable/core.js

```diff
@@ -244,4 +244,134 @@ QUnit.test( "nested resizable", function( assert ) {
 	outer.remove();
 } );
 
+QUnit.test( "Resizable with scrollbars and box-sizing: border-box", function( assert ) {
+	assert.expect( 4 );
+
+	$( "<style> * { box-sizing: border-box; } </style>" ).appendTo( "#qunit-fixture" );
+
+	//Both scrollbars
```

Comment:
Please start comments with a space. Please fix other comments as well:
```suggestion
	// Both scrollbars
```